### PR TITLE
Fix nginx config w/out symlink setup

### PR DESCRIPTION
**Pull request by NerdsvilleCEO**

### CODE REVIEW

**Comment by NerdsvilleCEO** [comment]
@nmadhok Could you review this PR, thanks!

**Comment by nmadhok** [comment]
@NerdsvilleCEO Reviewed and merged! Thanks!

File: jenkins/nginx.sls [hunk 1]
```diff
@@ -23,3 +25,18 @@ extend:
         - file: /etc/nginx/sites-available/jenkins.conf
       - require:
         - file: /etc/nginx/sites-enabled/jenkins.conf
+
+{% else %}
+
+Add nginx config for jenkins:
+  file.managed:
+    - template: jinja
+    - name: {{ jenkins.nginx_vhost_path }}/jenkins.conf
```
Comment (myoung34) [inline jenkins/nginx.sls]: this isnt in defaults.yml, is that on purpose?

Comment (NerdsvilleCEO) [inline jenkins/nginx.sls]: I didn't know how to handle that, it is a required value, I can set to default such as /etc/nginx/conf.d for example, but it is only going to run when symlink_vhost is False, not sure if it should be documented or assume a default

Comment (myoung34) [inline jenkins/nginx.sls]: Everything should assume a default, that way people can have a source of truth (defaults.yml) so they know what they can set

Comment (NerdsvilleCEO) [inline jenkins/nginx.sls]: Good to know, will add and make a PR :)

**Pull request merged**